cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -159,9 +159,9 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         }
       }
       section("Creating composite databases", "administration-databases-create-composite-database", "enterprise-edition") {
-        // TODO: add a link to the place explaining what composite databases are
         p(
-          """Composite databases do not contain data, but they reference to other databases that can be queried together through their constituent aliases.""".stripMargin)
+          """Composite databases do not contain data, but they reference to other databases that can be queried together through their constituent aliases.
+            |For more information about composite databases, see <<operations-manual#composite-databases-introduction, Composite database introduction>>.""".stripMargin)
```

Comment:
```suggestion
            |For more information about composite databases, see <<operations-manual#composite-databases-introduction, Operations Manual -> Composite database introduction>>.""".stripMargin)
```

Reply:
'→' surely?

Reply:
Looks fancier than -> , but ->  is what is used elsewhere, so unless you want to update all the other arrows throughout the manual, I think this should be -> .